src/identity/manager.rs

```diff
@@ -71,8 +71,10 @@ where
 }
 
 impl SecretManager<CaClient> {
-    pub fn new(cfg: crate::config::Config) -> SecretManager<CaClient> {
-        let caclient = CaClient::new(cfg.ca_address.unwrap(), cfg.auth);
+    pub fn new(cfg: Arc<crate::config::Config>) -> SecretManager<CaClient> {
+        let ca_address = cfg.ca_address.clone().unwrap();
+        let auth = cfg.auth.clone();
```

Comment:
Are these must be cloned?

Reply:
Hi @hzxuzhonghu,  yes, the ownership to the field of `Config` can not be moved into `CaClient`, so need to clone here. the same as the `ca_address`.